maintainer/travis/build_cmake.sh

```diff
@@ -96,12 +98,10 @@ if ! $insource; then
     if [ ! -d $builddir ]; then
         echo "Creating $builddir..."
         mkdir -p $builddir
+        cd $builddir
```

Comment:
Move this out of the `if [ ! -d $builddir ]`, otherwise the script cannot be safely rerun. It doesn't matter for the CI builds, but if somebody tries to run this script locally, it will fail on the second run.